### DISCUSSION TITLE
Any plan for windows support?

### DISCUSSION THREAD

**Discussion by metastag** [Taiko2k/Tauon#646] [Ideas]
Hey! I found out this music player just today and gave it a star after barely using it for 15 minutes!
Amazing cutomizability/looks + how fast it loaded my 6gb+ library :flushed: 
I was wondering will there ever be windows support? Groove Music is fine but i'd like to keep note of alternatives

(ps. windows support means am exponentially huge audience which could potentially multiply the popularity of this app)

**Comment by Taiko2k** [comment]
I used to support Windows but removed support. And since have drifted away from maintaining and considering support, so going back would be hard. And I'm fine with just supporting Linux. Im like committed now, no going back.

Just for discussion, I think the market in windows is kind of saturated, it's hard to gain popularity. Even in Linux which is growing, has less options for existing apps and much higher user engagement, gaining any market penitration has been very hard.

**Reply by Solarunit** [reply]
There are already many good music player on Windows, and I am glad that we finally have something so good on Linux.
But I wouldn't mind a cool new music player for Android  :)

**Reply by Taiko2k** [reply]
If I had kept working on Tauon Music Remote, it probably could have become a music app for Android. I gave up on that though.

If Linux phone takes off I might optimize Tauon for it. Can't imagine it taking off though, Linux phone is so damn rough currently.

**Reply by Solarunit** [reply]
I hope Tauon will stay a desktop music app. Not a fan of those 2-in-1 apps...

**Reply by Taiko2k** [reply]
Yeah, Its because of that I'd be inclined to have a separate touch mode UI, rather than trying to combine it into desktop.

**Reply by metastag** [reply]
Understandable, thanks for taking out the time to reply!

**Reply by RSKYS** [reply]
It's not the actual support, might not be very good solution:
You actually can run Tauon by WSLg cause it'd actually make unix stuffs possible,
ArchWSL, WSLackware(in progress) and.. are capable of proper installation from their user repositories;
Other distributions for WSLg are also fine if you struggle with flatpak a bit.

**Reply by Taiko2k** [reply]
>WSLg

I don't care to provide official support for WSL, but someone else could maintain a guide on how to do that. Ideally it would be comprehensive, easy to follow.
